src/ipahealthcheck/core/constants.py

```diff
@@ -6,15 +6,15 @@
 
 # Error reporting result
 SUCCESS = 0
-CRITICAL = 1
-ERROR = 2
-WARNING = 4
+WARNING = 10
+ERROR = 20
+CRITICAL = 40
 
 _levelToName = {
     SUCCESS: 'SUCCESS',
-    CRITICAL: 'CRITICAL',
-    ERROR: 'ERROR',
     WARNING: 'WARNING',
+    ERROR: 'ERROR',
+    CRITICAL: 'CRITICAL',
```

Comment:
Ala logging we should probably add since we're in here:

'CRITICAL': CRITICAL,
'ERROR': ERROR

etc to translate string to number.